declarations/Twitter.json

```
@@ -21,6 +21,123 @@
         ".p22-chapters-container",
         ".cr01-section"
       ]
+    },
+    "Community Guidelines - Self-harm": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/glorifying-self-harm",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Hate Speech": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/hateful-conduct-policy",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Child Sexual Exploitation": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/sexual-exploitation-policy",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Violent Incitement": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/glorification-of-violence",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Violent Organizations": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/violent-groups",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Spam": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/platform-manipulation",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Regulated Goods": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/regulated-goods-services",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Harassment": {
+      "fetch": "https://help.twitter.com/fr/rules-and-policies/abusive-behavior",
+      "select": ["#twtr-main"],
+      "remove": [
+        ".u11__breadcrumbs",
+        ".b24-share-tweet",
+        ".b01__headline.twtr-type--headline-md.inherit"
+      ]
+    },
+    "Community Guidelines - Intellectual Property": {
```

Seeing its contents, I feel this document describes _how copyright complaints (DMCA…) will be handled_.

```suggestion
    "Copyright Claims Policy": {
```